__tests__/__snapshots__/ordCdsrc.test.js.snap

```diff
@@ -5,11 +5,13 @@ exports[`Tests for default ORD document when .cdsrc.json is present Successfully
   "$schema": "https://sap.github.io/open-resource-discovery/spec-v1/interfaces/Document.schema.json",
   "apiResources": [
     {
-      "apiProtocol": "odata-v4",
-      "description": "AdminService",
+      "apiProtocol": "odata-v5",
```

Comment:
Why odata-v5? This would be an invalid choice / ORD document.

Reply:
I was thinking place a randomly value to distinguish test result, no special meaning. However, I have changed it in the coming commit.